scvi/model/_scanvi.py

```diff
@@ -172,7 +172,7 @@ def from_scvi_model(
         unlabeled_category
             Value used for unlabeled cells in `labels_key` used to setup AnnData with scvi.
         adata
-            AnnData object that has been registered via :func:`~scvi.data.setup_anndata`.
+            AnnData object that has been registered via :meth:`~scvi.model.SCANVI.setup_anndata`.
```

Comment:
@adamgayoso  Is this correct, or is it SCVI (or any other model)?

Reply:
pretty sure its SCANVI here